tests/test_builder_containerbuild.py

```diff
@@ -611,21 +612,23 @@ def test_osbs_build(self, tmpdir, pkg_info, failure, orchestrator):
             (flexmock(task)
                 .should_receive('_write_combined_log'))
 
+        build_args = {'git_branch': 'master'}
```

Comment:
I noticed that we started using inclusive terminology in our repos, should we also start using something other than "master" where possible?

Reply:
changed